README.md

```diff
@@ -66,19 +66,23 @@ Behaviour can be configured in the `app.json` under the `svgAppIcon` field. For
-| ---------------- | ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `foregroundPath` | `"./icon.svg"`            | Input file path for the foreground layer. File needs to exist, and may contain transparency.                                                                    |
-| `backgroundPath` | `"./icon-background.svg"` | Input file path for the background layer. File doesn't need to exist, and will default to a fully white background. If file exist, it needs to be fully opaque. |
-| `platforms`      | `["android", "ios"]`      | Array of platforms for which application launcher icons should be generated. Possible values are `android` and `ios`.                                           |
-| `force`          | `false`                   | When `true`, output files will always be written even if they are newer than the input files.                                                                   |
+| Field               | Default                                              | Description                                                                                                                                                     |
+| ------------------- | ---------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `foregroundPath`    | `"./icon.svg"`                                       | Input file path for the foreground layer. File needs to exist, and may contain transparency.                                                                    |
+| `backgroundPath`    | `"./icon-background.svg"`                            | Input file path for the background layer. File doesn't need to exist, and will default to a fully white background. If file exist, it needs to be fully opaque. |
+| `platforms`         | `["android", "ios"]`                                 | Array of platforms for which application launcher icons should be generated. Possible values are `android` and `ios`.                                           |
+| `force`             | `false`                                              | When `true`, output files will always be written even if they are newer than the input files.                                                                   |
+| `androidOutputPath` | `./android/app/src/main/res`                         | Where to place generated Android icons                                                                                                                          |
+| `iosOutputPath`     | `./ios/*/Images.xcassets/AppIcon.appiconset`         | Where to place generated iOS icons, defaults to first `Images.xcassets` found as a sub-sub-directory in `./ios`                                                 |
```

Comment:
Might be good to clarify that this should be an `.appiconset` directory. Now the description about the default might confuse someone that it should be an `.xcassets` directory.